src/box/lua/config/descriptions.lua

```diff
@@ -0,0 +1,2446 @@
+]])
+
+-- }}} process configuration
+
+-- {{{ replication configuration
+
+M['replication'] = format_text([[
+    This section defines configuration parameters related to replication.
+]])
+
+M['replication.anon'] = format_text([[
+    Whether to make the current instance act as an anonymous replica. Anonymous
+    replicas are read-only and can be used, for example, for backups.
+
+    To make the specified instance act as an anonymous replica, set
+    `replication.anon` to `true`.
+
+    Anonymous replicas are not displayed in the `box.info.replication` section.
+    You can check their status using `box.info.replication_anon()`.
+
+    While anonymous replicas are read-only, you can write data to
+    replication-local and temporary spaces (created with `is_local = true`
+    and `temporary = true`, respectively). Given that changes to
+    replication-local spaces are allowed, an anonymous replica might
+    increase the 0 component of the vclock value.
+
+    Here are the limitations of having anonymous replicas in a replica set:
+
+    - A replica set must contain at least one non-anonymous instance.
+    - An anonymous replica can't be configured as a writable instance by
+    setting database.mode to rw or making it a leader using
+    `<replicaset_name>.leader.`
+    - If `replication.failover` is set to election, an anonymous replica can
+    have `replication.election_mode` set to `off` only.
+    - If `replication.failover` is set to `supervised`, an external failover
+    coordinator doesn't consider anonymous replicas when selecting a bootstrap
+    or replica set leader.
+]])
+
+M['replication.bootstrap_strategy'] = format_text([[
+    Specifies a strategy used to bootstrap a replica set. The following
+    strategies are available:
+
+    - `auto`: a node doesn't boot if half or more of the other nodes in a
+    replica set are not connected. For example, if a replica set contains 2
+    or 3 nodes, a node requires 2 connected instances. In the case of 4 or 5
+    nodes, at least 3 connected instances are required. Moreover, a bootstrap
+    leader fails to boot unless every connected node has chosen it as a
+    bootstrap leader.
+    - `config`: use the specified node to bootstrap a replica set. To specify
+    the bootstrap leader, use the `<replicaset_name>.bootstrap_leader` option.
+    - `supervised`: a bootstrap leader isn't chosen automatically but should
+    be appointed using `box.ctl.make_bootstrap_leader()` on the desired node.
+    - `legacy` (deprecated since 2.11.0): a node requires the
+    `replication_connect_quorum` number of other nodes to be connected. This
+    option is added to keep the compatibility with the current versions of
+    Cartridge and might be removed in the future.
+]])
+
+M['replication.connect_timeout'] = format_text([[
+    A timeout (in seconds) a replica waits when trying to connect to a master
+    in a cluster. See orphan status for details.
```

Comment:
I guess that without a link this 'see' reference has no much sense.

Reply:
Dropped.